easybuild/easyconfigs/c/cget/cget-0.2.0-foss-2020b.eb

```
@@ -0,0 +1,34 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
+easyblock = 'PythonBundle'
+
+name = 'cget'
+version = '0.2.0'
+
+homepage = 'https://cget.readthedocs.io/en/latest/index.html'
+description = """Cmake package retrieval. This can be used to download and install cmake packages"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

I don't think this really needs to be at `foss` level, we can probably drop this to `GCCcore`
```suggestion
toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```